cmd/context/create.go

```diff
@@ -380,6 +380,17 @@ func (c ContextCommand) getUserContext(ctx context.Context, ns string) (*types.U
 				return nil, fmt.Errorf(oktetoErrors.ErrCorruptedOktetoContexts, config.GetOktetoContextsStorePath())
 			}
 		}
+
+		kubetoken, err := client.Kubetoken().GetKubeToken(okteto.Context().Name, okteto.Context().Namespace)
```

Comment:
Would it make sense to execute this query and the one we do to get the credentials from the backend in parallel to try to optimize it and not impact too much on the performance of every command executino?

Reply:
@ifbyol I've considered but didn't have time to try refactor, I think we're a bit tight this sprint to hold this now. I propose to have a follow up issue perhaps.

Reply:
If we do a follow up issue and we release this impacting performance is a worst experience than delaying the feature. This is a new request that is added in almos every command, we have to be careful about it

Reply:
@ifbyol I've already considered the implications and I believe it's better to keep the code simple instead of adding complexity here now. The call to get the kubetoken takes ~100 ms and it's only executing once at the beginning of a command. 

It's more relevant the performance of `okteto kubetoken` because if you call kubectl maby time, it will add up the latency